meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```diff
@@ -32,6 +35,28 @@ do_compile() {
         V=1
 }
 
+python do_prepare_mender_shell_conf() {
+    import json
+
+    # If a mender-shell.conf has been provided in SRC_URI, use this one instead
+    src_conf = os.path.join(d.getVar("WORKDIR"), "mender-shell.conf")
+    if os.path.exists(src_conf):
+        bb.debug(1, "mender-shell.conf already present in ${WORKDIR}, return.")
```

Comment:
Returning is not what you do though. And the behavior in the client is to still add the key, so I reckon we should do the same here. So in other words, only the message here is wrong.